task_11/css/style.css

```diff
@@ -0,0 +1,982 @@
+src: url('../fonts/ProximaNova-Semibold.woff') format('woff');
+font-weight: normal;
+font-style: normal;
+}
+
+html {
+  background-color: #eceef4;
+  font-family: "Proxima Nova Regular";
+  color: #a5aacb;
+}
+
+h1, h3 {
+  color: #6e768f;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+}
+
+i {
+  padding-right: 20px;
+  font-size: 20px;
+}
+
+button:focus, select:focus {
+  border: 1px solid #76a7e1;
+  border-radius: 5px;
+  outline: 0;
+}
+
+button:hover, select:hover, input:hover {
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+/*Header*/
+
+.header {
+  width: 90%;
+  margin: 20px auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 125px;
+  height: 125px;
+}
+
+.header-button {
+  width: 200px;
+  height: 65px;
+  position: relative;
+  background-color: #eceef4;
+  color: #a5abc5;
+  font-size: 15px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-align: center;
+  border: 1px solid #c9d0e1;
+  border-radius: 5px;
+}
+
+/*Main section class Order */
+
+.order {
+  background-color: #fff;
+  width: 100%;
+  max-width: 90%;
+  margin: 40px auto;
+  border: 2px solid #c9d0e1;
+  border-radius: 5px;
+}
+
+/*Summary*/
+
+.order-summary {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  border-bottom: 2px solid #c9d0e1;
+}
+
+.order-summary h3 {
+  font-weight: 700;
+  margin-bottom: 20px;
+}
+
+.order-summary i {
+  font-size: 28px;
+}
+
+/*Summary order*/
+
+.order-summary-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-grow: 2;
+  border-right: 2px solid #c9d0e1;
+}
+
+.summary-info {
+  display: flex;
+  flex-direction: column;
+  width: auto;
+  margin: 40px 0;
+  margin-left: 40px;
+}
+
+/*Summary table*/
+
+.summary-table {
+  width: 100%;
+  margin: 25px 5px 5px 25px;
+
+}
+
+.summary-table th {
+  color: #6e768f;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-align: left;
+  padding: 5px 20px;
+}
+
+/*Summary buttons*/
+
+.summary-buttons {
+  margin: 20px 0;
+  margin-right: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 25%;
+}
+
+.summary-buttons button {
+  position: relative;
+  width: 200px;
+  height: 65px;
+  margin: 10px auto;
+  font-size: 15px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-align: center;
+  border-radius: 5px;
+}
+
+.summary-buttons button[name="submit"] {
+  color: #fff;
+  background-color: #5fcac6;
+  border: 2px solid #54aca9;
+}
+
+.summary-buttons button[name="quote"] {
+  color: #a5abc5;
+  background-color: #f7f7fa;
+  border: 2px solid #d7d8e0;
+}
+
+/*Summary help*/
+
+.summary-help {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  margin: 40px 20px 10px 40px;
+}
+.summary-help a {
+  display: inline-block;
+  padding-top: 10px;
+  padding-right: 20px;
+  text-decoration: none;
```

Comment:
You may want adding `a:hover { text-decoration: underline; }` declaration to let user know that there are links and those are clickable.

Reply:
Thank for the rewiev!!!
Yes, these two are links, but I didn't show it because of two reasons:
1. these words are not underlined in the mockup
2.  I have used <a> tags just to improve mobile user experience: by tapping the phone number it will redirect user to dial that number.  The same thing is with email. And I haven't seen anywhere on the internet that the phone number or email address are ever underlined, have you? Well maybe emails sometimes are)))

Reply:
My suggestion was to add underlining on mouse hover event, just to give a hint on that these are clickable. Kinda when we add hover effect to buttons, which are per se clearly recognized as clickable objects without any added visual effect.
However, you are right, highlighting `tel:` might be appropriate for mobiles but mobile browsers do not offer any 'hover'.
It's absolutely up to you whether to change this or not.
Definitely won't be a blocker on approval from my side.

Reply:
Thank you! I will definitely use hover effects on some other a tags in future. Will keep it in mind)